.prettierignore

```diff
@@ -0,0 +1,4 @@
+/dist
+package-lock.json
```

Comment:
we shouldn't have package-lock if we are using yarn, I think that we could remove it

Reply:
you are right, removed